exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,22 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if(number < 0){
+    return true;
+  }else{
+    return false;
+  }
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  if(number < 0){
```

Comment:
This should calcuate if the number is between 0 and 10. This function would not work for 20, it will return true but it should be false.